acceptance-tests/basic_pcap_test.go

```diff
@@ -0,0 +1,68 @@
+package acceptance_tests
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+)
+
+var _ = Describe("Pcap Deployment", func() {
+	It("Deploys successfully", func() {
```

Comment:
We can have two test cases for that. 
The first one - "Pcap deploys successfully" which ensures only that the release can be deployed 
The second one - "Traffic captured successfully"

Reply:
The "can be deployed" part is tested implicitly with the `SynchronizedBeforeSuite` function. So I thought it makes little sense to then again test what we just tested and I added a little extra, also to keep the runtime low.

But sure, in the future we can add more detailed tests with different scenarios.

Reply:
Hm... you are right. But maybe it makes sense to make it more explicitly.  Especially for the test explicit case if deploy will not work. Then it will be very fast clear what use case is broken.